lib/cutting_edge/app.rb

```diff
@@ -35,6 +38,7 @@ def worker_fetch(repo)
 
 module CuttingEdge
    LAST_VERSION_TIMEOUT = 5
+   BADGE_WORK_QUEUE = 
```

Comment:
This is incomplete.

Reply:
Ah, thanks -- that's a remnant of me trying to implement a Queue, as per your other suggestion, and then realizing I had no idea how to. :)